README.md

```diff
@@ -21,130 +21,46 @@ under the License.
-The interface (gRPC) is between Device-Manager and user-application. Device-Manager also registers specific Redfish APIs for
-events from the device like alerts, removal/insertion events. It then publishes data on kafka bus to collect the data.
-
-# Co-Work with Edgecore PSME and OpenBMC Redfish
-```
-+-----------------------------+
-|   Edgecore Device Manager   |
-+-------------+-------------+-+
-    |        |          |
-    |        |          |
-+---v--+ +---v----+ +---v-----+
-| VOLT | | SWITCH | | OpenBMC |
-| PSME | |  PSME  | | Redfish |
-+------+ +--------+ +---------+
-```
-The Edgecore Device Manager could manage a wide range of devices (ex: Edgecore XGS-PON/GPON vOLTs, ONL/SONIC Switches, and OpenBMC device).
-Before you use the Device Manager, you have to install the PSME software to the device.
-More detailed information can be found at [DM-Redfish-PSME](https://github.com/opencomputeproject/DM-Redfish-PSME) and [DM-Redfish-OpenBMC](https://github.com/opencomputeproject/DM-Redfish-OpenBMC).
-
-# Preparation
-The Device Manager supports running the *Ubuntu Desktop 16.04 version*. And the hardware platform needs to match the minimum requirement,
-such as *CPU 2 cores/Memory 4GB and SSD driver 40GB*. The host system need to install necessary packages (ex: git, curl and docker)
+# Hardware requirements
+The Device Manager supports running the *Ubuntu Desktop 18.04 version*. For a single node deployment minimum requirement are
+- CPU 8 cores,
+- RAM 8GB,
+- Storage of at least 40GB.
 
 # Prerequisites
 Before building Device Manager, the following steps are required:
-- Go 1.17.10 
+- Go 1.17.10 installed,
   - To install Go and to download necessary packages together with libraries for building Device Manager, run:
   ```shell
   $ make go-install
   $ make prereq
   $ source ~/.bashrc
   ```
-- certificates must be generated (scripts to generate certificates are provided at https://github.com/ODIM-Project/ODIM/tree/main/build/cert_generator)
-- cert files must be placed in /etc/deviceManager/certs folder.
-
-## Install Device Manager and ODIM using single command
-This single command will install and run Device Manager as well as ODIM with all necessary packages.
+- Docker 20.10.18 (minimum version) installed,
+- Certificates delivered or generated and placed into (project root)/build/certs (scripts for generation can be found [here](https://github.com/ODIM-Project/ODIM/tree/main/build/cert_generator)),
+- Config changes to fit your needs:
+  - **Device Manager**
+    - (project root)/svc-device-manager/config/config.yml
+  - **ODIM services**
+    - (project root)/lib-utilities/config/odimra_config.json
+  - **Redis**
+    - (project root)/build/redis/redis.conf
+  - **Etcd**
+    - (project root)/build/etcd/etcd.yml
+
+## Install Device Manager using single command
+Use this command to install and run Device Manager together with ODIM services as Docker containers.
 
```

Comment:
Redis and etcd also as Docker containers.

Reply:
Fixed